server/channels/store/sqlstore/store.go

```diff
@@ -200,7 +200,7 @@ func New(settings model.SqlSettings, metrics einterfaces.MetricsInterface) (*Sql
 	store.stores.compliance = newSqlComplianceStore(store)
 	store.stores.session = newSqlSessionStore(store)
 	store.stores.oauth = newSqlOAuthStore(store)
-	store.stores.OutgoingOAuthConnection = NewSqlOutgoingOAuthConnectionStore(store)
+	store.stores.OutgoingOAuthConnection = newSqlOutgoingOAuthConnectionStore(store)
```

Comment:
The store name is also capitalized, while the others are not
```suggestion
	store.stores.outgoingOAuthConnection = newSqlOutgoingOAuthConnectionStore(store)
```